### ISSUE TITLE
How to solve response code 500 in cytargetlinker?

### ISSUE THREAD

**Issue by kristyccc** [CyTargetLinker/cytargetlinker#26]
Got a problem when I followed the tutorial 1 in cyTargetlinker. Every step is smooth until I reached step 4 after I selected "2" for the "overlap threshold" functionality. It gave a sign "Caught exception executing task" with a statement "Untitled (org.cytargetlinker.app.internal.tasks.FilterOverlapTask). I further checked the automation and it stated that there was an error 500 response code. Anyone know how to solve this problem?

![Screenshot 2020-04-23 at 2 28 10 PM](https://user-images.githubusercontent.com/64188223/80067209-01b9b300-8570-11ea-8812-f71f60d1548c.png)
![Screenshot 2020-04-23 at 2 28 38 PM](https://user-images.githubusercontent.com/64188223/80067221-09795780-8570-11ea-9114-3e04d3c15b35.png)
![Screenshot 2020-04-23 at 2 29 22 PM](https://user-images.githubusercontent.com/64188223/80067243-139b5600-8570-11ea-96a7-7f8e8a29c75d.png)

I used MacOS catalina 10.15.4 and 
download Cytoscape 3.8.0 (Java 11) and installed cyTargetLinker app v4.1.0.

Thanks.

**Comment by mkutmon** [comment]
Hey! I assume this might be related to the new Cytoscape release. I will try to check and fix asap.

**Comment by jessekowalski** [comment]
I am having the exact same issue and have the same macOS 10.15.4, Cytoscape 3.8.0 and cyTargetLinker 4.1.0.

**Comment by mkutmon** [comment]
Yeah, I can reproduce the error but still have to figure out why it happens :(. This takes more time than I expected. I will keep you posted. If you need the functionality now, you will need to go back to Cytoscape 3.7.2. Sorry about that.
